### PR TITLE
Romain

### PR DESCRIPTION
**Pull request by toolslive**
should fix ARAKOON-386

### CODE REVIEW

File: cfg/arakoon.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [global]
 #comma separated list of node names that are part of the cluster
-cluster = arakoon_0 ,arakoon_1, arakoon_2
+cluster = arakoon_0 ,arakoon_1
```
Comment (domsj) [inline cfg/arakoon.ini:3]: Is this change intentional?

Comment (toolslive) [inline cfg/arakoon.ini:3]: does it really matter?

**Comment by domsj** [comment]
If CI agrees then it LGTM.

File: cfg/arakoon.ini (hunk 1 above)
Comment (domsj) [inline cfg/arakoon.ini:3]: no
